modules/avfilter/util.c

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file util.c	 Video filter using libavfilter -- utility functions
+ *
+ * Copyright (C) 2020 Mikhail Kurkov
+ */
+
+#include <string.h>
+#include <re.h>
+#include <rem.h>
+#include <baresip.h>
+#include <libavutil/frame.h>
+#include "util.h"
+
+static int swap_lines(uint8_t *a, uint8_t *b, size_t size) {
+	uint8_t tmp[size];
```

Comment:
I think it is dangerous to declare a variable size array on the stack.

probably better to use a heap buffer, or a fixed size stack buffer